WordPress/Tests/Theme/DeregisterScriptUnitTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * A sniff unit test checks a .inc file for expected violations of a single
+ * coding standard. Expected errors and warnings are stored in this class.
+ *
+ * @category  Theme
+ * @package   WordPress_Coding_Standards
+ * @author    Simon Prosser <pross@pross.org.uk>
+ */
+class WordPress_Tests_Theme_DeregisterScriptUnitTest extends AbstractSniffUnitTest
+{
+
+	/**
+	 * Returns the lines where errors should occur.
+	 *
+	 * The key of the array should represent the line number and the value
+	 * should represent the number of errors that should occur on that line.
+	 *
+	 * @return array(int => int)
+	 */
+	public function getErrorList() {
+		return array(
+			3 => 0,
+			5 => 0,
```

Comment:
Generally speaking, in the WPCS project, this list would only contain the lines which actually have errors and their amount. You can safely remove the line numbers entries for those lines without errors.